src/Mod/Part/Init.py

```diff
@@ -32,4 +32,4 @@
 FreeCAD.addImportType("STEP with colors (*.step *.STEP *.stp *.STP)","Import")
 FreeCAD.addExportType("STEP with colors (*.step *.stp)","Import")
 
-FreeCAD.__unit_test__ += [ "TestPartApp" ]
+FreeCAD.__unit_test__ += [ "TestPartApp", "parttests.TopoShapeTest" ]

```

Comment:
IMO this addition should really be in TestPartApp.py, with the rest of the stuff in parttests -- what was the reason behind adding it here instead?

Reply:
TestPartApp.py is already large, and I would anticipate doubling its size with tests unless I go to additional files.  I think the parttests directory usage here is questionable, but trying to split TestPartApp.py up along object type & regression tests would seem to make sense.  See questions posed in the Toponaming chat.

This could certainly go into a TestPartAppTopoShape.py file next to TestPartApp.py for example.

Reply:
Hmm.  Maybe the approach in PartDesign is better;  it has the top level test, but that includes tests in a separate PartDesignTests directory.  Seems pretty clean.

Reply:
I seem to have tied myself in knots on this one.  In any case, removed the unnecessary \_\_unit_test__ addition, and it looks better now.